test/fixtures/WidenImage.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
```

Comment:
Interesting! Ya I can switch that. Was using...

    const ancestors = document.querySelectorAll("[id*='widthConstrainedAncestor']")

... but can just as easily use a class.